net-print/brother-mfc9340cdw-bin/brother-mfc9340cdw-bin-1.1.2.ebuild

```diff
@@ -1,47 +0,0 @@
-# Copyright 1999-2016 Gentoo Foundation
```

Comment:
Why was this file deleted?

Reply:
After updating an ebuild, we have to change its name. I added -r1 to ebuilds like brother-xxx1234-bin-1.0.ebuild and incremented those like brother-xxx1234-bin-1.0-r1.ebuild.
When multiple r1 r2 ebuilds were found, I kept only the last one and deleted older ones.